dhcpv4/server4/conn_linux.go

```diff
@@ -0,0 +1,45 @@
+package server4
```

Comment:
I have just noticed this is not linux specific so it can be renamed from `conn_linux` to `conn`